worldengine/cli/main.py

```diff
@@ -441,10 +450,10 @@ def main():
     if args.temps and len(args.temps.split('/')) is not 6:
         usage(error="temps must have exactly 6 values")
 
-    temps =[.874, .765, .594, .439, .366, .124]
+    temps = [.874, .765, .594, .439, .366, .124]
     if args.temps:
         temps = args.temps.split('/')
-        for x in range(0,6):
+        for x in range(0,len(temps)):
```

Comment:
I would suggest to add an assert to test the length of the array (otherwise a user might see crashes much later in the code and not understand them). Regarding the value-order etc. I would just leave that to the responsibility of the user. Should we start working on a global settings-file in the future, we could add a class to handle all kinds of stupid inputs.

(This also affects line 468.)